cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -1163,18 +1181,21 @@ instance FromJSON DelegationsAndRewards where
        pure (address, rewardAccountBalance, delegation)
 
 runQueryLeadershipSchedule
-  :: AnyConsensusModeParams
+  :: Maybe SocketPath
+  -> AnyConsensusModeParams
   -> NetworkId
   -> GenesisFile -- ^ Shelley genesis
   -> VerificationKeyOrHashOrFile StakePoolKey
   -> SigningKeyFile -- ^ VRF signing key
   -> EpochLeadershipSchedule
   -> Maybe OutputFile
   -> ExceptT ShelleyQueryCmdError IO ()
-runQueryLeadershipSchedule (AnyConsensusModeParams cModeParams) network
-                           (GenesisFile genFile) coldVerKeyFile (SigningKeyFile vrfSkeyFp)
-                           whichSchedule mJsonOutputFile = do
-  SocketPath sockPath <- lift readEnvSocketPath & onLeft (left . ShelleyQueryCmdEnvVarSocketErr)
+runQueryLeadershipSchedule
```

Comment:
Strange indent here. Did we hit the 100 char limit when you added `mNodeSocketPath`?

Reply:
I'm using the convention that arguments indent four spaces whilst the function implementation indents two.  This is to distinguish the two by ensuring they don't have the same indent.

Should I be using some other convention?